tests/acceptance/features/coreApiVersions/fileVersionAuthor.feature

```diff
@@ -389,11 +386,9 @@ Feature: file versions remember the author of each version
 
   Scenario: check the author of the file version (inside a folder) which was created before enabling the version storage
```

Comment:
Scenario can be removed as it needs some feature to be disabled and re-enabled, and there are no such steps that implement it.